### DISCUSSION TITLE
AsyncUDP is conflicting with AsyncTCP

### DISCUSSION THREAD

**Discussion by fester225** [community/community#114529] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

AsyncUDP conflicts with AsyncTCP. AsyncUDP has declarations for TCPIP_ADAPTER_IF_STA, TCPIP_ADAPTER_IF_AP, and TCPIP_ADAPTER_IF_ETH. I'd be interested in knowing why something supporting the UDP protocol involves variables that look like TCP. In any event I request the developers remove or change these reference names so they won't conflict with AsyncTCP.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @fester225 ! We're happy you're here. The Discussions category is solely related to conversations around the GitHub product Discussions. 


If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Async. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
